backend/src/routes.js

```diff
@@ -90,12 +95,12 @@ router.post('/ngos', jwtMiddleware({ secret: process.env.ACCESS_TOKEN_KEY, algor
         image,
         webpage,
         description,
-        main_representative,
+        mainRepresentative,
         affinities,
-        contact: { address, phone, contact_hours }
+        contact: { address, phone, contactHours }
```

Comment:
Same here, all json data coming from the frontend uses underscores in the variable names.